### ISSUE TITLE
v3 Extras DocLister popup window install throws 404 with custom manager name

### ISSUE THREAD

**Issue by nick0** [evolution-cms/evolution#2031]
(edit: I misread the version as 3.1.17 instead of 3.1.7)

Hello,

I have a Evolution 3.1.7 site with a custom manager name.

(if relevant - this site was updated from 1.4.15 to 3.1.7 with the Migration plugin and has DocLister 2.6.3 installed)

Not sure if this is an issue unique to my site or if this applies to all v3 sites with a custom manager folder name.

If I try to install any of the DocLister Optional Items via Extras, I get a 404 in the pop up window after clicking the 'INSTALL >' button.

![evo-3-1-17-extras---doclister-popup](https://user-images.githubusercontent.com/1768308/141264751-30c8c403-a8b3-4576-912a-dd8851c8baf0.jpg)

![evo-3-1-17-extras---doclister-404](https://user-images.githubusercontent.com/1768308/141264742-99f7ae05-cf51-4f66-8767-661bb5891ee5.jpg)

If I revert the custom-manager-name back to the default manager name, the DocLister install works as expected through the popup.

So it appears this is another issue in Evo 3 when using a custom manager name.

To replicate this issue:

- Use a custom manager folder name 
- Visit Extras
- Find DocLister and click the green Install button.
- In the pop up window select the relevant optional items and click the 'INSTALL >' button in the bottom RHS.
- A 404 is returned

Try the above again with the default manager name and the installation should work as expected,

Thanks and keep up the great work.

**Comment by fourroses666** [comment]
I think you are missing files. Can you duplicate the default manager folder and overwrite the css files ans see what happens?

**Comment by fourroses666** [comment]
I have a custom manager too*

I do remember that someone had that 404 too only when installing DocLister. 
Don't know the fix. Maybe installing from .zip in the Extras side panel works?

**Comment by fourroses666** [comment]
I did have issues with installing some Extras because Open BaseDir was active but I don't think thats your issue.

**Comment by nick0** [comment]
> I think you are missing files. Can you duplicate the default manager folder and overwrite the css files ans see what happens?

Thanks for the suggestion and ideas Marc. I uploaded the manager folder that comes with 3.1.17... no difference, I still get the 404 with the custom manager name. 
Ah interesting, lucky that yours works :)
I can manually ftp, it's not so much about getting the DocLister installed (I downloaded from another site and then ftp'd it in), it's about getting on top of all these little quirks in Evo 3 that do not exist in Evo 1.4.x.
I think this might be one for the devs.
